config/locales/en.yml

```diff
@@ -203,6 +202,44 @@ en:
         direct: Direct shared
   datetime:
     distance_in_words:
```

Comment:
Isn't this handled in the stuff Chris added in his PR: https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en-CA.yml

Reply:
Yup for the default scope they are handled. The UpdatedComponent was using 2 partial translations and joining them together and well they didn't play nicely together. So I added in an updated scope under the distance_in_words section for text to be displayed when a record is updated.

Reply:
Sounds good, thanks